mods/ts/rules/defaults.yaml

```diff
@@ -609,6 +609,17 @@
 	Tooltip:
 		Name: Palette
 
+^Tunnel:
```

Comment:
Identified the sprites as `WA01X` etc. but can't get them o work as easily by just pasting them:

![image](https://cloud.githubusercontent.com/assets/756669/9567864/e9d90392-4f39-11e5-9f66-7f1747cd38b4.png)

It results into a pixelish mess that probably has to be resolved differently.

Also in the future tunnel related logic traits might be attached to this.